src/coreclr/vm/codeman.cpp

```diff
@@ -1541,6 +1536,11 @@ void EEJitManager::SetCpuInfo()
 
 #if defined(TARGET_X86) || defined(TARGET_AMD64)
 
+    // Set Evex ISA based on instruction set present
```

Comment:
@khushal1996 This needs to be moved above `CPUCompileFlags.EnsureValidInstructionSetSupport();`. Can you please move this to [here](https://github.com/dotnet/runtime/pull/101938/files#diff-eba17a72b6caa8b52b196f3b84bfd20dfeaf39a89f410943a692d1f33f569f3bR1440)

Reply:
resolved this in my latest commit